### ISSUE TITLE
Toolbox: RotaryCraft Screwdriver non-functional

### ISSUE THREAD

**Issue by lossycrypt** [Dynious/RefinedRelocation#365]
Relevant mod versions:

```
RefinedRelocation-mc1.7.10-1.1.4
RotaryCraft 1.7.10 V9b
DragonAPI 1.7.10 V9b
```

Symptoms:
RoC Screwdriver can be put into and taken out of toolbox, but when it is the selected tool the overlay does not render (the toolbox icon doesn't show the screwdriver) and (shift-)right clicking any relevant blocks (engines, gearboxes) opens the blocks GUI like with an empty hand.

**Comment by blay09** [comment]
The overlay not rendering is something we can't easily fix. It happens because RotaryCraft doesn't use Vanilla's conventional methods for their item rendering, while we do.

For the other issues, I've fixed up the toolbox a bunch to work more correctly, so it should work better in the next version.

**Comment by lossycrypt** [comment]
As long as it'll work then i can live with a missing icon.

Btw, are you planning to also include mesuring tools like the Angular Transducer, Multimeter, etc? Those would certainly also benefit from having a box for them.

**Comment by ReikaKalseki** [comment]
> The overlay not rendering is something we can't easily fix. It happens because RotaryCraft doesn't use Vanilla's conventional methods for their item rendering, while we do.

It can be easily fixed. Get its IItemRenderer and call render() with that for RenderType inventory.

**Comment by blay09** [comment]
Right, will create a special item renderer for the toolbox and do it correctly.

**Comment by lossycrypt** [comment]
I tried this again with RefinedRelocation-mc1.7.10-1.1.12 and it still doesn't work.
The RoC Screwdriver does only render "in the hand" not on the toolbox inventory/hotbar icon.
I've also tried some other mods tools and while those do render on the icon non of them functions in the world. Tools i've found to not work at all: Carpenters Chisel+Hammer (Chisel2), EU-Reader (IC2), Electric Wrench (IC2). The TE Cresent Hammer only works with right-click but shift-right-click to dismantle is non-functional.
